packages/babel-standalone/src/plugins.js

```diff
@@ -0,0 +1,244 @@
+// @flow
+export { default as externalHelpers } from "@babel/plugin-external-helpers";
```

Comment:
Is it possible to auto-generate this file and the list in packages/babel-standalone/src/index.js?

Reply:
I will do that.

Reply:
Note that we shouldn't generate it from the list of the directories but we still need to maintain a list of included plugins (which wouldn't be duplicated anymore).